framework/helpers/logger/helper.yml

```diff
@@ -1,2 +1,4 @@
 frontscripts:
   - filename: logger.livecodescript
+libraries:
+  - filename: logger.livecodescript
```

Comment:
Because I'm using the libraryStack handler to initialize settings from the app.yml file.

Reply:
Hmm. We don’t want the script in the message path twice and it should be in front so that it gets early accsss to the messages. One option is to have a library script with the guts of the code and a script that is the frontscript which handles messages.

Reply:
I'll see if that has any other implications.  I think I was using a couple of script locals so that will have to change.

Reply:
I'm looking over the code now. I have `libraryStack` defined but it would not have been called since the script is used a `frontScript`. I think it would just be a matter of moving `logChanged`, `ulLogIt`, and `msgChanged` to a separate frontscript and using `logger.livecodescript` as a library. 

In the front script the code would just call a helper in `logger.livecodescript` that would determine whether or not to log.

```
# FrontScript Example
# LCB
on logChanged pMsg
  _loggerLogChanged pMsg
  if not the result then pass logChanged
end logChanged
````

```
# Library example
command _loggerLogChanged pMsg
  if sLogA["extensions"] AND sLogTarget is not empty then
    _log pMsg
   return true
  else
    return false
  end if
end _loggerLogChanged
```

Reply:
Hopefully I did this correctly.  I'm not clear on why you're putting those three events in frontscripts, but anyway it's done and will be in the next iteration.